arch/x86/kernel/cpu/resctrl/rdtgroup.c

```diff
@@ -3867,7 +3867,7 @@ void resctrl_offline_domain(struct rdt_resource *r, struct rdt_domain *d)
 
 	if (is_mbm_enabled())
 		cancel_delayed_work(&d->mbm_over);
-	if (is_llc_occupancy_enabled() && has_busy_rmid(r, d)) {
```

Comment:
this commit is missing the changes in rdtgroup_setup_default. It just changing some 0s to actual variables.

Reply:
@clsotog I have cleaned up.
the base kernel is [0].

[0], https://github.com/NVIDIA-BaseOS-6/linux-nvidia-6.5/tree/MPAM-dev